flink-core/src/main/java/org/apache/flink/api/common/io/GenericCsvInputFormat.java

```diff
@@ -314,6 +320,25 @@ protected void setFieldsGeneric(boolean[] includedMask, Class<?>[] fieldTypes) {
 		this.fieldIncluded = includedMask;
 	}
 
+	/**
+	 * Gets the Charset for the parser.Default is set to UTF-8
```

Comment:
Should we spell out `charset` as "character set" in the documentation? Also, space after period and missing period at end.